src/ontodia/data/sparql/provider.ts

```diff
@@ -17,6 +18,8 @@ const DEFAULT_PREFIX =
  PREFIX rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
  PREFIX owl:  <http://www.w3.org/2002/07/owl#>` + '\n\n';
 
+const HAS_MAIN_REPRESENTATION_URI = 'http://collection.britishmuseum.org/id/ontology/PX_has_main_representation';
```

Comment:
We don't need to have any ontology-related resource in source, code, please, move it do initialization of sparql provider in demo.